website/docs/r/jwt_auth_backend.html.md

```diff
@@ -71,7 +89,7 @@ The following arguments are supported:
 
 * `default_role` - (Optional) The default role to use if none is provided during login
 
-* `provider_config` - (Optional) Provider specific handling configuration
+* `provider_config` - (Optional) Provider specific handling configuration. All values should be a string, but the provider will convert to the appropriate type when configuring Vault.
```

Comment:
Since it seems like setting these parameters as non-strings does work, we could soften this language if we like:

```suggestion
* `provider_config` - (Optional) Provider specific handling configuration. All values may be strings, and the provider will convert to the appropriate type when configuring Vault.
```